ReleaseChecklist.md

```diff
@@ -20,50 +20,52 @@
 
 ### Create the Release
  - [ ] Create Github release page: https://github.com/ethereum/solidity/releases/new
- - [ ] On the release page, select the ``release`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
- - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e origin/release..origin/develop``).
- - [ ] Create a pull request from ``develop`` to ``release``, wait for the tests, then merge it.
+ - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
+ - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e v0.5.4..origin/develop``).
  - [ ] Make a final check that there are no platform-dependency issues in the ``solidity-test-bytecode`` repository.
- - [ ] Wait for the tests for the commit on ``release``, create a release in Github, creating the tag (click the `PUBLISH RELEASE` button on the release page.)
- - [ ] Wait for the CI runs on the tag itself (travis should push artifacts onto the Github release page).
+ - [ ] Check that all tests on the latest commit in ``develop`` are green.
+ - [ ] Click the `PUBLISH RELEASE` button on the release page, creating the tag.
+ - [ ] Wait for the CI runs on the tag itself (travis will push the source archive and the static linux binary onto the Github release page).
```

Comment:
So Travis pushes the source tarball now? Or was the just wrong? I don't see any change in Travis config and last time I asked you said the tarball is created manually...

Reply:
In the previous releases, I noticed that the tarball hash changed some time. Now I realized it was because it is pushed again by travis and thus just overwritten. We can change that once we drop travis altogether.

Reply:
ok. good to know.